tests/src/python/test_qgsproviderconnection_ogr_gpkg.py

```diff
@@ -158,6 +158,13 @@ def test_gpkg_fields(self):
         self.assertIn(table_info.primaryKeyColumns()[0], fields.names())
         self.assertEqual(fields.names(), ['fid', 'id', 'typ', 'name', 'ortsrat', 'id_long', 'geom'])
 
+        # aspatial table
+        fields = conn.fields('', 'myNewAspatialTable')
+        table_info = conn.table('', 'myNewAspatialTable')
+        self.assertFalse(table_info.geometryColumn())
+        self.assertIn(table_info.primaryKeyColumns()[0], fields.names())
+        self.assertEqual(fields.names(), ['fid'])
```

Comment:
If this is only for internal test, it's ok, but note that GPKG Standard doesn't define obligatory name 'fid' on attribute tables - [there is even an example table where it's defined as 'id'](https://www.geopackage.org/spec130/#gpkg_attributes_example).